### DISCUSSION TITLE
最近想明白了几个困扰自己多年的问题

### DISCUSSION THREAD

**Discussion by giscus** [edony-ink/comments#31] [General]
# 最近想明白了几个困扰自己多年的问题

想事情很深的人是很难迈出步伐，所有的事情都想的很多、很深，批判性太强，那么什么事情都不可能做到自己期待的结果，久而久之心态就是会变得悲观。悲观一定是正确的，但有的时候事情就是乐观的人做的。想要就争取，得到就珍惜，失去就宽心，没得到就是学到，把视野放远一点慢慢累积，虽然这样无法大富大贵，但至少可以有一个幸福简单的自在生活。

https://www.edony.ink/persoanl-problems-that-has-been-bothering-me-for-years/?utm_source=blogfinder

**Comment by 7wate** [comment]
尝试知行合一
